_includes/footer.html

```diff
@@ -1,59 +1,52 @@
-<div class="footer">
-    <div class="container">
-        <div class="row justify-content-between">
-            <div class="col-sm-7">
-                <img class="brand" src="assets/images/logo.svg">
+<footer class="footer">
+        <div class="row">
+            <div class="col-12 social d-flex justify-content-center">
```

Comment:
In my opinion we should just remove the TBD text and keep the same style. Maybe reduce a bit the height. But this position with icons of network in the center and together with the logo I think i s a bit strange.

Reply:
@dricazenck footer styles updated to mainly just fix the spacing above follow us for now as discussed.

Reply:
see new screenshot